app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormatExpr.scala

```diff
@@ -208,10 +282,114 @@ case object ChildBenefitNumber extends TextConstraint
 object TextConstraint {
   val defaultWholeDigits = 11
   val defaultFactionalDigits = 2
+  val defaultDisplayWidthSize = 20
+  val defaultDisplayWidthXsSizeForText = 4
+  val defaultDisplayWidthSSizeForText = 5
+  val defaultDisplayWidthMSizeForText = 10
+  val defaultDisplayWidthLSizeForText = 20
+  val defaultDisplayWidthXlSizeForText = 30
+  val defaultDisplayWidthXxlSizeForText = 40
+  val defaultDisplayWidthXsSizeForNumber = 2
+  val defaultDisplayWidthSSizeForNumber = 3
+  val defaultDisplayWidthMSizeForNumber = 4
+  val defaultDisplayWidthLSizeForNumber = 5
+  val defaultDisplayWidthXlSizeForNumber = 10
+  val defaultDisplayWidthXxlSizeForNumber = 20
+  val defaultSizeNumber = 10
```

Comment:
This seems not to be used

Reply:
Ok. I believe you are talking about `defaultSizeNumber` and `defaultSizePositiveNumber`. I mentioned in the above comment.